_projects/2023/100481433.Rmd

```diff
@@ -0,0 +1,279 @@
+---
+title: "The world is progressively embracing democracy"
+description: |
+  Chart Replication Project.
+  categories: "2023"
+author: Sofia Villamil Erosa
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+# Democracy
```

Comment:
Use level two headings `##` as the first level. Level one corresponds to the project title that is generated from the header of the file.